Not-CSharp.md

```diff
@@ -5,9 +5,9 @@ categories: resnet sample advanced ml nn
 excerpt_separator: <!--more--> 
 ---
 
-In Silicon Valley season 4 Jian-Yang builds an AI app, that identifies pictures of hotdogs.
+In Silicon Valley season 4, Jian-Yang builds an AI app that identifies pictures of hotdogs.
 Today I am going to make a similar app to identify programming languages from code fragments.
-Whichever one I use, the code tends to look like it's written in C#,
+Whichever fragment I use, the code tends to look like it's written in C#,
```

Comment:
Not sure if fragment is the right word here, but I want the language to be more specific than "one", since I was confused.

Reply:
must be "language", not "fragment"